picasso/src/main/java/com/squareup/picasso/Request.java

```diff
@@ -434,6 +436,29 @@ public Builder transform(Transformation transformation) {
       return this;
     }
 
+
+    /**
+     * Set a list of transformations to be applied to the image.  These transformations will
+     * overwrite any existing transformations.
```

Comment:
Do you have an example of when that would be useful?
On Feb 28, 2015 1:24 PM, "Jacob Tabak" <notifications@github.com> wrote:

> In picasso/src/main/java/com/squareup/picasso/Request.java
> <https://github.com/square/picasso/pull/912#discussion_r25561377>:
>
> > @@ -434,6 +436,29 @@ public Builder transform(Transformation transformation) {
> >        return this;
> >      }
> >
> > +
> > +    /**
> > +     * Set a list of transformations to be applied to the image.  These transformations will
> > +     * overwrite any existing transformations.
>
> I thought it was a bit odd that you can dump these transformations into
> the RequestCreator but you can't remove them or access them afterwards.
> This approach allows you to take an existing request creator and remove any
> transformations.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/square/picasso/pull/912/files#r25561377>.
>